soql-server-pg/src/main/scala/com/socrata/pg/server/analyzer2/Hashable.scala

```diff
@@ -124,6 +124,7 @@ object Hashable {
           hasher.hashByte(9)
           hasher.hash(lim)
           hasher.hash(off)
+        case rewrite.Pass.RemoveOrderBy => hasher.hashByte(10)
```

Comment:
Feels like this is from a different working branch.

Reply:
(I added it because I'm worried about jenkins accidentally pulling it in and failing the buld because this project has error-on-warn turned on)

Reply:
ah